react/index.d.ts

```diff
@@ -2149,6 +2158,9 @@ declare namespace React {
         unselectable?: boolean;
     }
 
+    interface HTMLAttributes<T> extends HTMLAttributesBase<T, EventTarget> {
```

Comment:
instead of adding a bunch of `XXXBase` interfaces, maybe we can do something like:

```ts
interface ChangeTargetAttributes<T extends HTMLElement> extends HTMLAttributes<T> {
    onChange?: ChangeEventHandler<T> & { target: T };
}
```

Thoughts?